book.user.adoc

```diff
@@ -0,0 +1,86 @@
+= ownCloud User Manual
+The ownCloud Developers <docs@owncloud.com>
```

Comment:
User - developer ?

Reply:
Maybe "The ownCloud Team" would be more universal.

Reply:
👍 Good idea, on all three locations btw...